src/email-helpers.js

```diff
@@ -54,21 +54,22 @@ module.exports = async function sendEmail(filename, url, sender, recipient, tran
 
   // send email
   return new Promise((success, fail) => {
-      transporter.sendMail(mailOptions, function (err, info) {
-        if (err) {
-          spinner.fail('Error sending email' + err);
-          fail(err);
-          exit(1);
-        } else {
-          spinner.succeed('Email sent successfully');
-          deleteTemporaryImage(emailbody);
-          success(info);
-        }
-      });
+    transporter.sendMail(mailOptions, function (err, info) {
```

Comment:
This might be a lot more clear using a try..catch.. .and if necessary a throw on self-detected error (err value is not null?)